sonar-ws/src/main/gen-java/org/sonarqube/ws/QualityProfiles.java

```diff
@@ -0,0 +1,2483 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: ws-qualityprofiles.proto
+
+package org.sonarqube.ws;
+
+public final class QualityProfiles {
+  private QualityProfiles() {}
+  public static void registerAllExtensions(
+      com.google.protobuf.ExtensionRegistry registry) {
+  }
+  public interface WsSearchResponseOrBuilder extends
+      // @@protoc_insertion_point(interface_extends:sonarqube.ws.rules.WsSearchResponse)
+      com.google.protobuf.MessageOrBuilder {
+
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    java.util.List<org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile> 
+        getProfilesList();
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile getProfiles(int index);
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    int getProfilesCount();
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    java.util.List<? extends org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfileOrBuilder> 
+        getProfilesOrBuilderList();
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfileOrBuilder getProfilesOrBuilder(
+        int index);
+  }
+  /**
+   * Protobuf type {@code sonarqube.ws.rules.WsSearchResponse}
+   *
+   * <pre>
+   * WS api/qualityprofiles/search
+   * </pre>
+   */
+  public static final class WsSearchResponse extends
+      com.google.protobuf.GeneratedMessage implements
+      // @@protoc_insertion_point(message_implements:sonarqube.ws.rules.WsSearchResponse)
+      WsSearchResponseOrBuilder {
+    // Use WsSearchResponse.newBuilder() to construct.
+    private WsSearchResponse(com.google.protobuf.GeneratedMessage.Builder<?> builder) {
+      super(builder);
+      this.unknownFields = builder.getUnknownFields();
+    }
+    private WsSearchResponse(boolean noInit) { this.unknownFields = com.google.protobuf.UnknownFieldSet.getDefaultInstance(); }
+
+    private static final WsSearchResponse defaultInstance;
+    public static WsSearchResponse getDefaultInstance() {
+      return defaultInstance;
+    }
+
+    public WsSearchResponse getDefaultInstanceForType() {
+      return defaultInstance;
+    }
+
+    private final com.google.protobuf.UnknownFieldSet unknownFields;
+    @java.lang.Override
+    public final com.google.protobuf.UnknownFieldSet
+        getUnknownFields() {
+      return this.unknownFields;
+    }
+    private WsSearchResponse(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "WsSearchResponse" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)